plugins/lsp/language-servers/default.nix

```
@@ -553,6 +553,12 @@ with lib; let
       settingsOptions = import ./rust-analyzer-config.nix lib pkgs;
       settings = cfg: {rust-analyzer = cfg;};
     }
+    {
+      name = "slint_lsp";
+      description = "slint_lsp for Slint GUI language";
+      serverName = "slint-lsp";
+      package = pkgs.slint-lsp;
+    }
```

```suggestion
    {
      name = "slint-lsp";
      description = "slint_lsp for Slint GUI language";
      serverName = "slint_lsp";
    }
```

No, actually, you need to do this change.